tests/test_structlog.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+from typing import Callable, Iterator
+
+import pytest
+import structlog
+from structlog.stdlib import LoggerFactory
+
+from logot import Logot, logged
+from logot.structlog import StructlogCapturer
+
+logger = structlog.get_logger()
+
+
+@pytest.fixture
+def stdlib_logger() -> Iterator[None]:
+    structlog.configure(logger_factory=LoggerFactory())
```

Comment:
What is this needed for?

Reply:
By default, structlog loggers do not have a name - this only happens when using the stdlib, where you can pass a name as a positional parameter. See https://www.structlog.org/en/stable/configuration.html#logger-factories

Reply:
Can you achieve the same thing by using `bind(name="foo")`?

Reply:
I'm struggling to entirely follow the docs on this one

Reply:
Worth mentioning, the "name" here is open to a bit of interpretation. In stdlib logging it's the logger name. In loguru it's the __name__ of the module where the log was made (which is generally the same thing). So for structlog it would be good to get as close to that meaning as possible.

The intent is to filter a subset of the logs for a given package. For example, in `dave.module` tests I set the name to "dave" or "dave.module" to only capture logs from my lib, and not 3rd party

Reply:
`bind(name="foo")` isn't the same thing. Structlog wraps an underlying logger and uses that to output the logs. This can be a structlog / twisted / `logging` -defined logger, or something else. 

There is also the provided [CallsiteParameterAdder](https://www.structlog.org/en/stable/api.html#structlog.processors.CallsiteParameterAdder) processor, which will add this information to the event dict. The `module` added with this would map most closely to the meaning you've described, but we can't rely on it being part of the processor chain.

For now, I've left it as taking the `name` attribute of the underlying logger.

This might be a good example of something to pass as additional configuration to the `Capturer` as mentioned above, or to the which could be done in a general way to allow users of the lib to match anything added to the `event_dict`.